modules/dr-restoring-cluster-state.adoc

```diff
@@ -85,7 +85,7 @@ $ sudo mv /etc/kubernetes/manifests/kube-apiserver-pod.yaml /tmp
 +
 [source,terminal]
 ----
-$ sudo crictl ps | grep kube-apiserver | egrep -v "operator|guard"
+$ sudo crictl ps | grep kube-apiserver | egrep -v "operator|guard" | grep -v guard
```

Comment:
Not sue if we need to pass grep -v guard again ? As we are covering that in egrep.

Reply:
Ah, ok! Will remove

Reply:
Updated :)